content/en/agent/basic_agent_usage/windows.md

```diff
@@ -58,9 +58,12 @@ When the install finishes, you are given the option to launch the Datadog Agent
 
 1. Open PowerShell with **Administrator** privileges.
 2. Run the following command to install the Datadog Agent:
-    ```powershell
-    Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-agent-7-latest.amd64.msi APIKEY="<YOUR_DATADOG_API_KEY>"'
-    ```
+    {{< code-block lang="powershell" >}}
+$p = Start-Process -Wait -PassThru msiexec -ArgumentList '/qn /i datadog-agent-7-latest.amd64.msi /log C:\Windows\SystemTemp\install-datadog.log APIKEY="<DATADOG_API_KEY>"'
```

Comment:
Do you know why we run the command against a local msi in this doc while we use `https://s3.amazonaws.com/ddagent-windows-stable/datadog-agent-7-latest.amd64.msi` in the command provided by fleet? 
IMO the s3 url is a much nicer experience.